### ISSUE TITLE
[Bug]: Script not working with Ubuntu 22 LTS

### ISSUE THREAD

**Issue by NaXal** [Angristan/OpenVPN-install#1026]
### Make sure your check these beforehand!

- [X] Issues - https://github.com/angristan/openvpn-install/issues
- [X] README and FAQ - https://github.com/angristan/openvpn-install
- [X] Wiki - https://github.com/angristan/openvpn-install/wiki
- [X] Discussions - https://github.com/angristan/openvpn-install/discussions

### Server OS

Ubuntu 22 LTS

### OpenVPN version

2.5.0

### Client

Ubuntu 22 LTS

### What is the bug?

Hello,

I have used this script flawlessly with Ubuntu 20 LTS images deployed in AWS / Linode / DigitalOcean. It still works flawlessly but with Ubuntu 22 LTS images deployed at those locations, this script is not starting the OpenVPN after reboot.

Server is not showing OpenVPN NIC or the server IP address

Further, status of OpenVPN is showing "active (exited)"

![02](https://user-images.githubusercontent.com/91847371/185988972-59e99e0b-a3a1-407c-8497-6816d8f7af8c.jpg)

How do I fix this? 

Please help troubleshooting it.

Thanks.

### Relevant log output

```shell
C:\Windows\system32>ssh root@xxx.xxx.xxx.130
root@xxx.xxx.xxx.130's password:
Welcome to Ubuntu 22.04.1 LTS (GNU/Linux 5.15.0-46-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

  System information as of Mon Aug 22 11:27:17 PM IST 2022

  System load:           0.46630859375
  Usage of /:            13.6% of 24.04GB
  Memory usage:          16%
  Swap usage:            0%
  Processes:             102
  Users logged in:       0
  IPv4 address for eth0: xxx.xxx.xxx.130
  IPv6 address for eth0: xxx:xxx::fxxxc:9xxx:fxxxc:9xxx


0 updates can be applied immediately.


Last login: Mon Aug 22 23:23:14 2022 from xxx.xxx.xxx.158
root@localhost:~# systemctl status openvpn
● openvpn.service - OpenVPN service
     Loaded: loaded (/lib/systemd/system/openvpn.service; enabled; vendor preset: enabled)
     Active: active (exited) since Mon 2022-08-22 23:26:23 IST; 6min ago
    Process: 626 ExecStart=/bin/true (code=exited, status=0/SUCCESS)
   Main PID: 626 (code=exited, status=0/SUCCESS)
        CPU: 1ms

Aug 22 23:26:23 localhost systemd[1]: Starting OpenVPN service...
Aug 22 23:26:23 localhost systemd[1]: Finished OpenVPN service.
root@localhost:~#
```

**Comment by zefir-git** [comment]
Same here.

```
Consider setting groups/curves preference with tls-groups instead of forcing a specific curve with ecdh-curve.
Note: Treating option '--ncp-ciphers' as  '--data-ciphers' (renamed in OpenVPN 2.5).
Options error: --ca fails with 'ca.crt': No such file or directory (errno=2)
Options error: --cert fails with 'server_nWei2rYEpgaWYlVd.crt': No such file or directory (errno=2)
WARNING: cannot stat file 'server_nWei2rYEpgaWYlVd.key': No such file or directory (errno=2)
Options error: --key fails with 'server_nWei2rYEpgaWYlVd.key': No such file or directory (errno=2)
Options error: --crl-verify fails with 'crl.pem': No such file or directory (errno=2)
Options error: Please correct these errors.
Use --help for more information.
openvpn@server.service: Main process exited, code=exited, status=1/FAILURE
```

**Comment by AR45H** [comment]
Any fix for this issue?

**Comment by Lboylphd** [comment]
Got the same Problem

**Comment by angristan** [comment]
Duplicate of https://github.com/angristan/openvpn-install/issues/1000

**Comment by zefir-git** [comment]
> Duplicate of #1026

this is #1026

**Comment by angristan** [comment]
Sorry, I meant https://github.com/angristan/openvpn-install/issues/1000

**Comment by impishlucy** [comment]
defenetly not fixed if i still get that in 2024 on 22.04
